lib/auth/oauth.dart

```diff
@@ -1,16 +1,16 @@
 part of dart_openapi;
-// @dart=2.9
+
```

Comment:
i think we should remove nullability from this - you shouldn't be allowed to create one with a null access token.